src/Commands/field/FieldEntityReferenceHooks.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+namespace Drush\Commands\field;
+
+use Drupal\Core\Entity\EntityTypeBundleInfoInterface;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drush\Commands\DrushCommands;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Question\ChoiceQuestion;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+class FieldEntityReferenceHooks extends DrushCommands
+{
+    use EntityTypeBundleValidationTrait;
+
+    public function __construct(
+        protected EntityTypeManagerInterface $entityTypeManager,
+        protected EntityTypeBundleInfoInterface $entityTypeBundleInfo,
+    ) {
+    }
+
+    public static function create(ContainerInterface $container): static
+    {
+        return new static(
+            $container->get('entity_type.manager'),
+            $container->get('entity_type.bundle.info'),
+        );
+    }
+
+    /**
+     * @hook on-event field-create-field-storage
```

Comment:
these should be attributes

Reply:
Do you know which attribute exactly? I can't seem to find an `OnEvent` attribute.